crates/blockifier/src/execution/sierra_utils.rs

```diff
@@ -239,3 +246,25 @@ pub fn create_callinfo(
         accessed_storage_keys: HashSet::new(),
     })
 }
+
+pub fn encode_str_as_felts(msg: &str) -> Vec<Felt> {
```

Comment:
I would add some unit-tests here, also why we can't use just 
```rust
let data = msg.as_bytes();
let chunks = data.chunks(CHUNK_SIZE); 
<...>
```
and also we can use `hex` crate to convert easily

Reply:
Well because I didn't now about Chunks. Definetly looks better with it

Regarding `hex` create I don't see any benefits to be truthful.

Reply:
Regarding unit tests, I agree, but I don't want to put time into there until I've tested the solutions with the different syscalls and it's functional.

I've already made an issue to adding tests for Sierra Utils (not just this, but most of the functions defined there)

Reply:
hex would be better because we won't rewrite anything twice

Reply:
I haven't used the `hex` crate before. Could you explain the logic steps of how using `hex` would be better so I can learn